libvast_test/artifacts/schemas/base.schema

```diff
@@ -0,0 +1,7 @@
+// This file contains common types and aliases.
```

Comment:
Why is this file duplicated here? That sounds like a risk in terms of maintenance.